docs: add mention about executor args in generate crds (#5)
kubeshop/testkube-argocd: README.md

kubectl patch configmaps -n argocd argocd-cm --patch-file customization/argocd-plugins.yaml
```

> If there is need to pass executor arguments to the test, it can be achieved adding ```--executor-args``` to the testkube comand in customization/argocd-plugin.yaml 

Reviewer: ```suggestion
If there is need to pass executor arguments to the test, it can be achieved by adding ```--executor-args``` to the testkube comand in customization/argocd-plugin.yaml.
```

> If there is need to pass executor arguments to the test, it can be achieved adding ```--executor-args``` to the testkube comand in customization/argocd-plugin.yaml 

Reviewer: ```suggestion
If there is need to pass executor arguments to the test, add ```--executor-args``` to the Testkube command in customization/argocd-plugin.yaml.
```
